ci/ray_ci/doc/api.py

```diff
@@ -19,3 +26,64 @@ class API:
     name: str
     annotation_type: AnnotationType
     code_type: CodeType
+
+    def from_autosummary(doc: str, current_module: Optional[str] = None):
```

Comment:
add return type? why it can return an empty list? maybe add doc string on what does it return?

I thought `from_xxx` should work like a constructor and return `API`?

Reply:
it returns itself yes, in this case though, it can return List[API], or in the other function, Optional[API]